internal/cli/internal/command/deploy/deploy.go

```diff
@@ -151,7 +119,7 @@ func run(ctx context.Context) error {
 		tb := render.NewTextBlock(ctx, fmt.Sprintf("Release command detected: %s\n", releaseCommand.Command))
 		tb.Done("This release will not be available until the release command succeeds.")
 
-		if err := watchReleaseCommand(ctx, releaseCommand.ID); err != nil {
+		if err := watch.WatchReleaseCommand(ctx, releaseCommand.ID); err != nil {
```

Comment:
`watch.WatchReleaseCommand` doesn't read all that great - maybe `watch.ReleaseCommand` would be better I think